tests/word2vec_lookup.cc

```diff
@@ -0,0 +1,25 @@
+#include <memory>
+#include <vector>
+
+#include "finalfusion-cxx/Embeddings.hh"
+#include "finalfusion.h"
+
+int main() {
+  try {
+    std::unique_ptr <Embeddings>
```

Comment:
No space between the type and template arguments. Or does `clang-format` do this?

Reply:
Not `clang-format` but CLion auto formatting, I'll fix this later